### PR TITLE
SAK-46023 Date Manager > Nullable fields for some activities needs to be allowed

### PR DESCRIPTION
**Pull request by gwickram**
https://sakaiproject.atlassian.net/browse/SAK-46023

### CODE REVIEW

File: site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java
Comment (jesusmmp) [inline site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java:462]: Please use rb.getString with a property for i18n

File: site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java
Comment (jesusmmp) [inline site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java:471]: Please use rb.getString with a property for i18n

File: site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java
Comment (gwickram) [reply site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java]: Moved the error messages to properties file.

File: site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java
Comment (gwickram) [reply site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java]: Moved the error messages to properties file.

**Review by ottenhoff — APPROVED** [review]
We made a few edits while on the core call to improve formatting.

Let's wait until this passes the build tests and then merge it in!

File: site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java
Comment (ottenhoff) [inline site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java:539]: would prefer these two lines to be on one line per the conventions above

File: site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java
Comment (ottenhoff) [inline site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java:632]: I would prefer less line breaks here.

**Pull request merged**
